src/test/java/com/google/api/codegen/testdata/python_doc_types_library.baseline

```diff
@@ -0,0 +1,65 @@
+============== file: google/cloud/proto/example/library_v1/types.py ==============
```

Comment:
@geigerj maybe you have an idea.

I implemented it using the gapic package name but that produced:
google/cloud/**gapic**/example/library_v1/

Now I'm thinking to construct it from the package config shortname name and version but I am trying to figure out how to get the base path.
as in
{base/path}/{shortname}_{version}

Thoughts?